lib/framework/data_context.dart

```diff
@@ -857,6 +857,9 @@ class APIResponse with Invokable {
   @override
   Map<String, Function> getters() {
     return {
+      'success': () => _response?.isSuccess,
```

Comment:
rule of thumb is use the same prefix for variables that are related to one another. Just use the exact name is*

Reply:
Done